### DISCUSSION TITLE
Exposición «Sistemas operativos mínimos para la nube»: Preguntas y respuestas

### DISCUSSION THREAD

**Discussion by gwolf** [unamfi/sistop-2022-1#179] [Q&A]
Abro este foro como un espacio de discusión para la exposición que prepararon los alumnos Alberto García Figueroa Munguía y Edgar Cristóbal García Gutiérrez, «Sistemas operativos mínimos para la nube». El video está disponible en:

https://www.youtube.com/watch?v=y5wQ1-foFlQ

Pueden consultar también el documento de texto en que los alumnos presentan este mismo tema:

https://github.com/unamfi/sistop-2022-1/blob/main/exposiciones/GarciaFigueroaAlberto-GarciaGutierrez/Sistemas_Operativos_Minimos_para_la_nube.pdf

Además de participar aquí, les pido que llenen la pequña encuesta que está disponible en:

https://www.questionpro.com/t/ARiwuZqIqp

**Comment by JesusESD** [comment]
¡Saludos compañeros! Muchas felicidades por su exposición, la verdad me agradó mucho. Lo único que no me quedó muy claro fue ¿por qué no existen los bloqueos mutuos en el OSv? En el video dan una breve explicación, pero honestamente no lo entendí del todo. ¿Podrían ayudarme con eso? Les agradeceré mucho su respuesta :)

**Comment by Inakyord** [comment]
Hola, espero se encuentren bien. El tema se me hizo interesante, pero algo complejo. Una de las preguntas que se me ocurrieron al tomar en cuenta el hipervisor que permite cargar varios sistemas operativos en un mismo hardware (y por lo que entendí de varios usuarios diferentes), fue: ¿El hipervisor es parte de los sistemas operativos en la nube, ó firmware del hardware? ¿En algún momento un sistema operativo ejecutándose sabe de la existencia del otro con los mismos recursos ó el hipervisor los engaña? Gracias.

**Comment by CarlosGMendozaB** [comment]
Hola, me pareció muy interesante el tema. ¿Conocen o saben si en el sistema operativo para la nube, existen tipos de nube, o solo esta definida una? Y si existe, ¿En características tienen y para quien esta dirigido?. Gracias.
